xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css

```diff
@@ -179,13 +169,6 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
 tbody.xwiki-livetable-display-body tr {
   border-bottom: 1px solid $theme.borderColor;
 }
-tbody.xwiki-livetable-display-body tr:hover , tbody.xwiki-livetable-display-body tr.rowHover { /* for IE6 tr:hover fix*/
```

Comment:
Thanks for putting back "thead.xwiki-livetable-display-header tr.xwiki-livetable-display-filters:hover" too, and sorry for missing these. Without all the IE6 hacks, the code is much more understandable now. Thank you!